--- lib/viewvc.py.config	2011-11-03 22:32:34.000000000 +0800
+++ lib/viewvc.py	2011-11-04 09:35:02.876221107 +0800
@@ -4451,10 +4451,7 @@
   # Try to find the configuration pathname by searching these ordered
   # locations: the environment, the passed-in PATHNAME, the hard-coded
   # default.
-  pathname = (env_pathname
-              or pathname
-              or os.path.join(os.path.dirname(os.path.dirname(__file__)),
-                              "viewvc.conf"))
+  pathname = '/etc/viewvc/viewvc.conf'
 
   # Load the configuration!
   cfg = config.Config()
--- conf/viewvc.conf.dist.config	2011-09-15 01:54:10.000000000 +0800
+++ conf/viewvc.conf.dist	2011-11-04 09:33:42.355758980 +0800
@@ -691,7 +691,7 @@
 ## This allows ViewVC to present discrete pages to the users instead of
 ## the entire directory.  Set to 0 to disable pagination.
 ##
-#dir_pagesize = 0
+dir_pagesize = 150
 
 ## log_pagesize: Maximum number of revision log entries on a given page.
 ## This allows ViewVC to present discrete pages to the users instead of
